tests/build/test_docker.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from ebonite.build.docker import image_exists_at_dockerhub, repository_tags_at_dockerhub
+from ebonite.utils.module import get_python_version
+from tests.build.conftest import has_docker
+
+
+@pytest.mark.docker
+@pytest.mark.skipif(not has_docker(), reason='no docker installed')
+def test_image_exists():
+    assert image_exists_at_dockerhub(f'python:{get_python_version()}-slim')
+    assert image_exists_at_dockerhub('minio/minio:latest')
+    assert image_exists_at_dockerhub('postgres:alpine')
+    assert image_exists_at_dockerhub('registry:latest')
+
+
+@pytest.mark.docker
+@pytest.mark.skipif(not has_docker(), reason='no docker installed')
+def test_image_not_exists():
+    assert not image_exists_at_dockerhub(f'python:100.200.300')
+    assert not image_exists_at_dockerhub('ebonite:forever')
+    assert not image_exists_at_dockerhub('minio:minio')
```

Comment:
it's not clear that those images really do not exist))) i cases like this i prefer to use names like "minio:this_tag_definitely_does_not_exist_ololo"

Reply:
Do not see any difference here between your and my examples:
1. Python: it is at version 3 after 20 years, so version 100 is unrealistic
1. Minio: it seems unreasonable to use package name as its version, thus unrealistic
1. Ebonite: There is no such images at all and we fully control this

Reply:
the difference is not that this could be real images, the difference is that when i see "minio:minio", i need to think about it, and when i see "minio:this_does_not_exist" i do not

Reply:
ok, I've got your point, fixed